user/details/script.js

```diff
@@ -661,6 +664,197 @@ function showContent() {
   section3.classList.remove('hide');
 }
 
+function generateRemainingDays(dateStr) {
+  const date = new Date(dateStr);
+  const now = new Date();
+  const offset = 330 * 60 * 1000;
+  const nowInIST = new Date(now.getTime() + offset);
+  const diff = nowInIST - date;
+  if (diff >= 24 * 60 * 60 * 1000) {
+    return Math.floor(diff / (24 * 60 * 60 * 1000)) + ' days ago';
+  } else if (diff >= 60 * 60 * 1000) {
+    return Math.floor(diff / (60 * 60 * 1000)) + ' hours ago';
+  } else if (diff >= 60 * 1000) {
+    return Math.floor(diff / (60 * 1000)) + ' minutes ago';
+  } else {
+    return Math.floor(diff / 1000) + ' seconds ago';
+  }
+}
+
+function formatDate(dateStr) {
+  const date = new Date(dateStr);
+  const options = {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+    hour: '2-digit',
+    minute: '2-digit',
+    timeZone: 'Asia/Kolkata',
+  };
+  const formattedDate = date.toLocaleDateString('en-US', options);
+  return formattedDate.split(',')[0];
+}
+
+function generatePRsTabDetails() {
+  const div = createElement({
+    type: 'div',
+    classList: ['hidden-content', 'hide'],
+  });
+  const prsSec = createElement({
+    type: 'div',
+    classList: ['user-pr'],
+  });
+  const pagination = createElement({ type: 'div', classList: ['pagination'] });
+  const prevBtn = createElement({
+    type: 'button',
+    classList: ['pagination-load-more'],
+  });
+  prevBtn.appendChild(createTextNode('Load More'));
+  prevBtn.addEventListener('click', loadMorePRs);
+  pagination.append(prevBtn);
+  div.append(prsSec, pagination);
+  document.querySelector('.accordion-prs').appendChild(div);
+}
+
+async function getUserPRs() {
+  try {
+    const res = await makeApiCall(
+      `${API_BASE_URL}/pullrequests/user/${username}`,
+    );
+    if (res.status === 200) {
+      const data = await res.json();
+      userAllPRs = data.pullRequests;
+      totalPRsPages = Math.ceil(userAllPRs.length / PRsPerPage);
+      const prs = getPRstoFetch(userAllPRs, currentPageIndex);
+      generatePRsTabDetails();
+      generateUserPRsList(prs);
+    }
+  } catch (err) {}
+}
+
+function getPRstoFetch(userPR, currentIndex) {
+  const startIndex = (currentIndex - 1) * PRsPerPage;
+  return userPR.slice(startIndex, startIndex + PRsPerPage);
+}
+
+function generateUserPRsList(userPRs) {
+  if (!userPRs.length) {
+    const errorE1 = createElement({
+      type: 'p',
+      classList: ['error'],
+    });
+    const container = document.querySelector(
+      '.accordion-prs > .hidden-content',
+    );
+    container.innerHTML = '';
+    errorE1.appendChild(createTextNode('No PRs to show'));
```

Comment:
For all other accordion we have used `No data Found`. We can be consistent with that.

Reply:
done, i have created the then call it